bears/general/LineLengthBear.py

```diff
@@ -16,7 +16,7 @@ class LineLengthBear(LocalBear):
     def run(self,
             filename,
             file,
-            max_line_length: int=80,
+            max_line_length: int=79,
```

Comment:
\+ this needs adaptions in coala, as we break this rule sometimes. Would be cool if you open another PR in coala fixing the line lengths everywhere :D